### PR TITLE
1.11 Dependencies Bugfix

### PR DESCRIPTION
**Pull request by mlodi-hqs**
* Fixed dependencies issues

### CODE REVIEW

File: qoqo/src/operations/analog_operations.rs [hunk 1]
```diff
@@ -21,8 +21,6 @@ use roqoqo::operations::*;
 use struqture::spins::SpinHamiltonian;
 use struqture_py::spins::SpinHamiltonianSystemWrapper;
 
-#[cfg(feature = "json_schema")]
-use roqoqo::ROQOQO_VERSION;
```
Comment (nfwvogt) [inline qoqo/src/operations/analog_operations.rs:25]: Are we sure this is not needed even with the json_schema feature?

File: roqoqo/Cargo.toml [hunk 2]
```diff
@@ -22,24 +22,26 @@ path = "src/lib.rs"
...
-rand = { version = "0.8.4"}
-async-trait = {version = "0.1", optional = true}
-futures ={version = "0.3", optional=true}
-petgraph = {version = "0.6.2", optional=true}
-bincode = {version="1.3", optional=true}
-struqture = {version = "1.5"}
+schemars = { version = "0.8", optional = true }
+jsonschema = { version = "0.17", optional = true }
+rand_distr = { version = "0.4", optional = true }
+rand = { version = "0.8.4" }
```
Comment (nfwvogt) [inline roqoqo/Cargo.toml:35]: maybe "~0.8" would be better here

File: roqoqo/src/operations/mod.rs [hunk 3]
```diff
@@ -813,37 +813,37 @@ pub trait OperateMultiQubitGate:
 }
 
 /// Marker trait to show that some operation has been implemented in roqoqo 1.1.0
-pub(crate) trait ImplementedIn1point1: Operate {}
+pub trait ImplementedIn1point1: Operate {}
```
Comment (nfwvogt) [inline roqoqo/src/operations/mod.rs:816]: Was that change required by linting?

**Review by nfwvogt — APPROVED** [review]

File: roqoqo/src/operations/mod.rs (hunk 3 above)
Comment (mlodi-hqs) [reply roqoqo/src/operations/mod.rs:816]: `rustc` warned it was never used, I'll revert and add a `#[allow(dead_code)]`

File: qoqo/src/operations/analog_operations.rs (hunk 1 above)
Comment (mlodi-hqs) [reply qoqo/src/operations/analog_operations.rs:25]: `rustc` flags it as unused import, testing the feature shows no problems

**Review by kbarkhqs — APPROVED** [review]
